docs/source/distributed.rst

```diff
@@ -52,12 +52,19 @@ MPI supports CUDA only if the implementation used to build PyTorch supports it.
 Backends that come with PyTorch
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-PyTorch distributed currently only supports Linux. By default, the Gloo and NCCL backends
+PyTorch distributed currently only supports Linux and Windows. By default for Linux, the Gloo and NCCL backends
```

Comment:
"PyTorch distributed currently only supports Linux and Windows." -> "PyTorch distributed package supports Linux (stable), MacOS (stable), and Windows (prototype)."